packages/core/src/components/button/abstractButton.tsx

```diff
@@ -62,7 +62,7 @@ export interface IButtonProps<E extends HTMLButtonElement | HTMLAnchorElement =
     large?: boolean;
 
     /**
-     * If set to `true`, the button will display a centered loading spinner instead of its contents.
+     * If set to `true`, the button will display a centered loading spinner instead of its contents, and the button will be disabled.
```

Comment:
we don't actually set the `disabled` attribute, so I think this change is more appropriate:

```suggestion
     * If set to `true`, the button will display a centered loading spinner instead of its contents, and the button will appear disabled.
```

Reply:
@adidahiya According to [this line](https://github.com/palantir/blueprint/blob/758233c54de4bb707a76131507938a11d7d72e1b/packages/core/src/components/button/abstractButton.tsx#L121), the button does actually get disabled when `loading` is set. Am I reading it wrong?

Reply:
hmm wait, you are right. sorry about that, will revert my edit!

Reply:
Awesome thanks!